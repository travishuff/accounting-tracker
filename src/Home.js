import React from 'react'

import './css/home.css'
import banana from '../images/banana-logo.png'

const Home = () => {
  return (
    <div className="hero-container">
      <div className="app-name">Banana Tracker</div>
      <img src={banana} alt="Banana Tracker logo" />
    </div>
  )
}

export default Home
