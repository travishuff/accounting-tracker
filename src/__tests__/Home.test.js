import React from 'react'
import { create } from 'react-test-renderer'

import Home from '../Home'

// Snapshot test
test('snapshot', () => {
  const c = create(<Home />)

  expect(c.toJSON()).toMatchSnapshot()
})
