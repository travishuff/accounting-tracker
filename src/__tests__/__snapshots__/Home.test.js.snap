// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="hero-container"
>
  <div
    className="app-name"
  >
    Banana Tracker
  </div>
  <img
    alt="Banana Tracker logo"
    src={Object {}}
  />
</div>
`;
